src/plugins/data/common/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -337,16 +337,18 @@ export class IndexPattern implements IIndexPattern {
    * @param field
    */
   getFormatterForField(
-    field: IndexPatternField | IndexPatternField['spec'] | IFieldType
+    field: IndexPatternField | IndexPatternField['spec'] | IFieldType,
```

Comment:
I don't think we should allow overriding field format params on the index pattern level, that opens up a lot of new possibilities.

Reply:
could we investigate how to override uiSetting on the server ? if you could do something like:

```
 uiSettings.override('timezone', myStoredTimezone);
 const table = tabify(await searchSource.fetch());
```

would that solve the problems ?